### PR TITLE
Fix EditText Validation

### PR DESCRIPTION
**Pull request by hamza-vd**
Apply validation for empty spaces. See this [issue](https://github.com/opensrp/opensrp-client-anc/issues/656)

### CODE REVIEW

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/customviews/NativeEditText.java [hunk 1]
```diff
@@ -156,7 +156,7 @@ public boolean validate() {
         }
 
         CharSequence text = getText();
-        boolean isEmpty = text.length() == 0;
+        boolean isEmpty = text.toString().trim().length() == 0;
```
Comment (ellykits) [inline android-json-form-wizard/src/main/java/com/vijay/jsonwizard/customviews/NativeEditText.java:159]: How about using the `StringUtils#isEmpty `method from the apache common library?

Comment (ellykits) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/customviews/NativeEditText.java:159]: You can have it check for `isBlank` which will look for white spaces so you do not have to trim

**Review by ellykits — CHANGES_REQUESTED** [review]

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/customviews/NativeEditText.java (hunk 1 above)
Comment (hamza-vd) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/customviews/NativeEditText.java:161]: Replaced with `StringUtils.isBlank(text)`

**Review by ellykits — APPROVED** [review]
:+1: LGTM

**Pull request merged**